### DISCUSSION TITLE
开启插件时打开B站直播间，会自动弹出翻译对话框

### DISCUSSION THREAD

**Discussion by masteryi12580** [the1812/Bilibili-Evolved#1359] [综合讨论 / General]
开启插件时打开B站直播间，会自动弹出翻译对话框
![image](https://user-images.githubusercontent.com/74101420/102990866-144b1880-4553-11eb-890c-85b05797cd00.png)

**Comment by the1812** [comment]
![image](https://user-images.githubusercontent.com/26504152/102990989-4eb4b580-4553-11eb-9a7c-e37255456dcd.png)
这里应该可以选择不再提示的吧

**Reply by masteryi12580** [reply]
是可以，但是每次进直播间都会弹出来，每次都要手动关闭，但我把插件关了就没有这种情况了。

**Reply by the1812** [reply]
Edge 选了不再提示还会弹出来?

**Reply by masteryi12580** [reply]
我刚刚已经选了“不再翻译此站点”，已经好了。但是我试着关闭插件后弹窗就没有了，所以才怀疑是插件的问题。抱歉。。。
